packages/backend/src/middlewares/jwtAuthMiddleware/jwtAuthMiddleware.ts

```diff
@@ -2,21 +2,26 @@
 // This rule is failing in CI but passes locally
 import {
     ForbiddenError,
+    JWT_HEADER_NAME,
     NotFoundError,
     ParameterError,
 } from '@lightdash/common';
 import { NextFunction, Request, Response } from 'express';
 import * as Account from '../../auth/account';
-import { decodeLightdashJwt, JWT_HEADER_NAME } from '../../auth/lightdashJwt';
+import { decodeLightdashJwt } from '../../auth/lightdashJwt';
 import { EmbedService } from '../../ee/services/EmbedService/EmbedService';
 import Logger from '../../logging/logger';
 
 /**
  * We don't have the parsed routes yet, so we get the path params in a
  * cheap and dirty way. Long-term, we'll use org-level JWTs rather than project-level.
  */
-const parseProjectUuid = (path: string) => {
-    const pathParts = path.split('/');
+const parseProjectUuid = (req: Request) => {
```

Comment:
perhaps 

```suggestion
const parseProjectUuid = (req: Pick<Request, 'query' | 'path'>) => {
```

Reply:
in case we want to reuse it in some other places where request object is not available. Not a blocker.